Classes/Issues/EditTitle/EditIssueTitleViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  EditIssueTitleViewController.swift
+//  Freetime
+//
+//  Created by B_Litwin on 10/22/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import GitHubAPI
+import SnapKit
+
+protocol EditIssueTitleViewControllerDelegate: class {
+    func sendEditTitleRequest(newTitle: String, viewController: EditIssueTitleViewController)
+    var currentIssueTitle: String? { get }
+    var viewerCanUpdate: Bool { get }
+}
+
+class EditIssueTitleViewController: UIViewController {
+
+    private let textView = UITextView()
+    private let issueTitle: String
+    private weak var delegate: EditIssueTitleViewControllerDelegate?
+    
+    init(delegate: EditIssueTitleViewControllerDelegate) {
+        self.delegate = delegate
+        self.issueTitle = delegate.currentIssueTitle ?? ""
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        preferredContentSize = CGSize(
+            width: Styles.Sizes.contextMenuSize.width,
+            height: 120
+        )
+        title = NSLocalizedString("Edit", comment: "")
```

Comment:
String `"Edit"` [already exists](https://github.com/GitHawkApp/GitHawk/blob/c23945ac7dfe015f3f7218668b5caef2355ac7db/Classes/Issues/Comments/IssueCommentSectionController.swift#L119), mind tossing that in `Constants.swift`?